README.md

```diff
@@ -93,46 +95,65 @@ Migrations
 ### Migrations to 0.80
 
 0.80 introduces a completely different way to manage tags for events.
+
+It is very important that you test this migration in a pre-production environment as once you drop the materialized view
+and tag columns you can not roll back.
+ 
+Most of the migration can be done while the old version of your application is running. The remaining steps happen automatically
+when a persistent actor is recovered.
```

Comment:
Is there any overhead to that, should users expect the recovery to take longer than "normal"?

Reply:
I guess it can if there were a lot of missed events, but not by much, i'll add a note